### PR TITLE
Rename interceptor in manifest and add a better description

### PR DESCRIPTION
**Pull request by KillariDev**

### CODE REVIEW

File: app/manifestV2.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"name": "Interceptor",
-	"description": "Interceptor",
+	"name": "The Interceptor",
+	"description": "Gain a clearer understanding of your Ethereum transactions with The Interceptor browser extension",
```
Comment (jubalm) [inline app/manifestV2.json:3]: Offering a description suggestion

```suggestion
	"description": "Know exactly what is happening with your Ethereum transactions without putting anything at risk",
```

File: app/manifestV3.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-	"name": "Interceptor",
-	"description": "Interceptor",
+	"name": "The Interceptor",
+	"description": "Gain a clearer understanding of your Ethereum transactions with The Interceptor browser extension",
```
Comment (jubalm) [inline app/manifestV3.json:3]:
```suggestion
	"description": "Know exactly what is happening with your Ethereum transactions without putting anything at risk",
```

**Review by jubalm — APPROVED** [review]
I feel that the "benefit" aspect of the tool would be important to the description being "risk-free"

Maybe @MicahZoltu would like to leave his thoughts as well before we merge it

File: app/manifestV2.json (hunk 1 above)
Comment (MicahZoltu) [reply app/manifestV2.json:3]:
```suggestion
	"description": "Gain a clearer understanding of your Ethereum transactions before any of your assets are at risk.  Simulate a series of transactions to see what will happen before signing anything.  Get warned of common scams and mistakes before you become a victim of them.",
```

**Pull request merged**